components/kserve/feature_templates.go

```diff
@@ -0,0 +1,28 @@
+package kserve
+
+import (
+	"embed"
+	"io/fs"
+	"path"
+)
+
+//go:embed templates
+var kserveEmbeddedFS embed.FS
+
+const baseDir = "templates"
+
+var Templates = struct {
+	// ServiceMeshDir is the path to the Service Mesh templates.
+	ServiceMeshDir string
+	// InstallDir is the path to the Serving install templates.
+	InstallDir string
+	// GatewaysDir is the path to the Serving Istio gateways templates.
+	GatewaysDir string
+	// Files the templates to be used
+	Files fs.FS
```

Comment:
I am not sure if `.Files` is the right name. `.Location` or even `.Source`. WDYT?

Reply:
I think `.Source` could be best :+1: